frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature

```diff
@@ -130,3 +130,121 @@ Feature: Create the pipeline from builder page
               And user clicks Create button on Pipeline Builder page
              Then user will be redirected to Pipeline Details page with header name "pipeline-workspace"
               And user will see workspace mentioned as "git-opt (optional)" in the Workspaces section of Pipeline Details page
+
+
+        @regression, @manual
+        Scenario: Add finally task node
+            Given user is at Pipeline Builder page
+             When user clicks on Add finally task
+              And user clicks on Select task
+              And user select "git-cli"
+              And user clicks on Add finally task again
+              And user clicks on Select task
+              And user selects "git-clone"
+             Then user sees Add finally task bubble is attached to the Select Task
+              And user sees "git-cli" and "git-clone" tasks in parallel
+              And user sees "Add finally task" option below "git-clone" task
+
+
+        @regression
+        Scenario: Create a pipeline with finally task node
+            Given user is at Pipeline Builder page
+             When user enters pipeline name "pipeline-finally"
+              And user selects the first task as "git-clone"
+              And user clicks on Add finally task
+              And user selects "git-clone" from Select task list
+              And user clicks on Add finally task again
+              And user selects "openshift-client" from Select task list
+              And user clicks on Add finally task again
+              And user selects "kn" from Select task list
+              And user fills task details
+              And user clicks on Create
+             Then user will be redirected to Pipeline Details page with header name "pipeline-params"
```

Comment:
user entering pipeline as "pipeline-finally" and here the header name is "pipeline-params" I think this is incorrect

Please check the spaces, looks like there is double spaces